cmd/earthly/app/run.go

```diff
@@ -57,7 +57,7 @@ func (app *EarthlyApp) unhideFlags() error {
 	var err error
 	if os.Getenv("EARTHLY_AUTOCOMPLETE_HIDDEN") != "" && os.Getenv("COMP_POINT") == "" { // TODO delete this check after 2022-03-01
 		// only display warning when NOT under complete mode (otherwise we break auto completion)
-		app.BaseCLI.Console().Warnf("Warning: EARTHLY_AUTOCOMPLETE_HIDDEN has been renamed to EARTHLY_SHOW_HIDDEN\n")
+		app.BaseCLI.Console().Warnf("%s", "Warning: EARTHLY_AUTOCOMPLETE_HIDDEN has been renamed to EARTHLY_SHOW_HIDDEN\n")
```

Comment:
We don't need format:
```suggestion
		app.BaseCLI.Console().Warn("Warning: EARTHLY_AUTOCOMPLETE_HIDDEN has been renamed to EARTHLY_SHOW_HIDDEN\n")
```

Reply:
Gemini said the same. 😄 Good spot!

The `ConsoleLogger` does not have `Warn()`, and I assumed it is a 3rd-party package. So can't do anything but use `Warnf()`.

Upon a second check, I can actually add `Warn()`, `Debug()`, etc.